StorageUtility/Linux/Device.cpp

```diff
@@ -15,90 +15,40 @@ See the License for the specific language governing permissions and
 limitations under the License.
 </License>
 */
+
 #include "Device.h"
 #include "StorageUtility.h"
 
 namespace vtStor
 {
 
-/*
-cDevice::cDevice(const PSP_DEVINFO_DATA& DevInfoData, const PSP_DEVICE_INTERFACE_DATA& DevInterfaceData, const PSP_INTERFACE_DEVICE_DETAIL_DATA& DevDetailData, U32 SizeOfDevDetailData)
+cDevice::cDevice(String DevicePath, String SysDevicePath)
 {
-    AllocateMemories(DevInfoData, DevInterfaceData, DevDetailData, SizeOfDevDetailData);
-    CopyMemories(DevInfoData, DevInterfaceData, DevDetailData, SizeOfDevDetailData);
+    this->m_DevicePath = DevicePath;

```

Comment:
please don't use "this->", just use directly the variable instead, it makes code longer seeing